### PR TITLE
move Logic to aggregation item tranlsate pipe

### PR DESCRIPTION
**Pull request by rjs-picturepark**

### CODE REVIEW

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/shared-module/pipes/aggregation-item-translate.ts [hunk 1]
```diff
@@ -2,16 +2,20 @@ import { Pipe, PipeTransform, Injectable } from '@angular/core';
 
 // LIBRARIES
 import { AggregationResultItem } from '@picturepark/sdk-v1-angular';
+import { TranslationService } from '@picturepark/sdk-v1-angular-ui';
```
Comment (neotek) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/shared-module/pipes/aggregation-item-translate.ts:5]: Import relative

Comment (rjs-picturepark) [reply src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/shared-module/pipes/aggregation-item-translate.ts]: Done

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/shared-module/pipes/aggregation-item-translate.ts [hunk 2]
```diff
@@ -2,16 +2,20 @@ import { Pipe, PipeTransform, Injectable } from '@angular/core';
...
+  constructor(private translationService: TranslationService) {}
 
   transform(item: AggregationResultItem, locale: string): string {
-    if (item) {
-      return item.getDisplayName(locale);
+    const displayName = item.getDisplayName(locale);
+
+    if (displayName === '') {
+      return this.translationService.translate('AggregationItemTranslatePipe.noValue');
+    } else {
```
Comment (rjs-picturepark) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/shared-module/pipes/aggregation-item-translate.ts:17]: remove else

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/utilities/translations.ts [hunk 3]
```diff
@@ -75,6 +75,12 @@ export let TRANSLATIONS = {
       de: 'Keine Filter konfiguriert',
     },
   },
+  AggregationItemTranslatePipe: {
+    noValue: {
+      en: 'No value',
+      de: 'kein wert',
```
Comment (rjs-picturepark) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/utilities/translations.ts:81]: upper case  k and w

**Review by neotek — APPROVED** [review]

**Review by neotek — APPROVED** [review]

**Pull request merged**